### ISSUE TITLE
Small formatting errors and some suggestions for MCM doc.

### ISSUE THREAD

**Issue by Sentrus** [ixanza/xanzas_lotd#1]
Complete Alchemy -> Basic Needs
Hunger Penalties are already disabled by default. Should probably say "Enable Fatigue Penalties" instead.

Complete Crafting
Bulleted list cleanup - Need a new line before going to Crafting Options section. Same for Learning and XP.

Diverse Dragons Col. 3
Add note to user to exit MCM after configuring this mod and wait for "DDC3: ACCM Splicing has completed!" message in the top left. (About 20 seconds)

Easy Wheel

Farmhouse Chimneys
Suggest rewording Arthmoor Villages section to "Enable ALL EXCEPT Falkreath" or similar to make it more succinct and so Falkreath being disabled doesn't get lost in the list.

Frostfall
More specifically tell user to wait for "(Done! Frostfall is enabled.) message in top left after Enabling and exiting MCM before re-entering and loading profile.
Current Profile is already set to LEXY LOTD but I had to re-select it to get things to load properly. Might be worth specifying so users don't see its already set and not click it.

Immersive Armours
Instruct user to exit MCM and wait for "Immersive Armors Configuration Has Finished" (About 30 seconds)

Immersive Creatures
    Creature toggles: Most of these are toggled off already in your list, Problem pre-disabling Goblin?

iWant RND Widgets
    Since this requires activation of RND first, you should move this instruction down below RND activation instruction.

Legacy of the Dragonborn
    since LOTD Settings is a top-level MCM already, you can probably adjust the bullet points here to be LOTD Settings -> General -> Shipment Crate LOcations

Predator Vision
Need a new line after color boost instruction, same for after Sound 2

Timing is Everything
This may require more fiddling than suggested. I had to load preset more than twice to get all the settings to stick. Perhaps add a couple of values for users to check to see if it completed properly. (LIke vampire attacks at 30 instead of 8)

Thieves Guild Requirements
Loading it once, exiting, and reentering, and loading it again didn't work for me. I had to click it twice in the same MCM session to get it to load properly. Perhaps add a user check to see one or two of the values are correct (I had Sneak requirement to join at 125 when it was improperly loaded, for example)

Trade and Barter
Barter Rate setting is actually called Barter Presets

Ultimate Combat
Hardcore Damage and Stealth are already disabled. PLayer Killmove immunity already enabled.

Wildcat Combat
Need newline after Disabled Injuries.

**Comment by ixanza** [comment]
Thanks. Addressed all of these. Decided to go back to manual setup for Timing is Everything and Thieves Guild Requirements. Not worth the hassle getting the presets to work.
